### PR TITLE
BigFix: Correct indices in me.pyx

### PR DESCRIPTION
**Pull request by alongd**
### Motivation or Problem
The `chemically-significant eigenvalues` ME method isn't functioning due to a bug in index mix-up between `u` and `v` introduced in the RMG Py 3 transition of `me.pyx`.

### Description of Changes
Indices in `me.pyx` corrected according to the pre-Py 3 transition version.

### Testing
Added a simple functional test for the CSE method.

### Reviewer Tips
Run the CSE example on the master branch to see the bug. Run it on this branch to see that it runs to completion. See that the simple code modifications make sense.

### CODE REVIEW

**Review by amarkpayne — COMMENTED** [review]
Thanks for catching this! I have some quick comments on the PR. I didn't get a chance to review the example commit yet

File: rmgpy/pdep/cse.pyx [hunk 1]
```diff
@@ -108,17 +108,17 @@ def apply_chemically_significant_eigenvalues_method(network, list lumping_order=
     for r in range(n_rows):
         for s in range(r):
             if me_mat[r, s] != 0:
-                if abs(me_mat[r, s] - me_mat[s,r]) > 0.01 * me_mat[r, s]:
-                    if me_mat[r, s] > 1e-200 or me_mat[s,r] > 1e-200:
-                        print(r, s, me_mat[r, s], me_mat[s,r])
+                if abs(me_mat[r, s] - me_mat[s, r]) > 0.01 * (me_mat[r, s] or me_mat[s, r]) \
+                    and (me_mat[r, s] > 1e-200 or me_mat[s, r] > 1e-200):
+                        print(r, s, me_mat[r, s], me_mat[s, r])
```
[the remark this answers is not in the document]
Comment (alongd) [reply rmgpy/pdep/cse.pyx]: Excellent suggestion. Added as a fixup

**Comment by alongd** [comment]
@amarkpayne Can you take a look at the rmg-tests? They're not behaving, I don't think it's related to this branch (also fail on #2066).
Please let me know when to squash this branch. Thanks

**Comment by amarkpayne** [comment]
This looks good to me, go ahead and squash and I'll merge it in (I'll check with the other developers what our policy is right now with Travis being down)

**Comment by alongd** [comment]
Thanks, @amarkpayne ! PR squashed :)

**Review by amarkpayne — DISMISSED** [review]
After talking with @mjohnson541 we are holding off on merging in PRs for the time being to see if we can fix the RMG-Tests issue. I'll try to stay on top of this PR so that it gets merged in when we are ready. Thanks for the PR @alongd !

**Comment by alongd** [comment]
@amarkpayne, rmg-tests pass now, I think this PR is good to go.

**Review by mjohnson541 — APPROVED** [review]

**Pull request merged**